config/locales/views/documents.en.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  documents:
+    edit:
+      document_edit: Edit your file
```

Comment:
`'documents.edit.document_edit'` is a bit redundant.
Try `documents.edit.title`, as it's actually a title